SPECS/azurelinux-image-tools/azurelinux-image-tools.spec

```diff
@@ -93,12 +93,17 @@ go test -C toolkit/tools ./...
 %license LICENSE
 %{_bindir}/imagecustomizer
 # Container support files - placed in container filesystem paths with imagecustomizer- prefix
-/usr/local/bin/imagecustomizer-entrypoint.sh
-/usr/local/bin/imagecustomizer-run.sh
-/usr/local/bin/telemetry_hopper.py
-/imagecustomizer-telemetry-requirements.txt
+%{_prefix}/local/bin/imagecustomizer-entrypoint.sh
+%{_prefix}/local/bin/imagecustomizer-run.sh
+%{_prefix}/local/bin/telemetry_hopper.py
+%{_sysconfdir}/imagecustomizer-telemetry-requirements.txt
```

Comment:
Placing this under `/etc` doesn't seem right to be. I think this should be `/usr/lib/imagecustomizer/telemetry-requirements.txt`